operator/controllers/suite_test.go

```diff
@@ -96,7 +96,7 @@ var _ = BeforeSuite(func() {
 		WaitForChartReadiness: false,
 	}
 	k8sClientFromManager = k8sManager.GetClient()
-	reconciler.SetHardDeleteTimeout(hardDeleteTimeout)
+	HardDeleteTimeout = hardDeleteTimeout
```

Comment:
We also need to adjust hard delete check interval for tests. Default value is set too high for tests `hardDeleteTimeout`, so I suggest to set `HardDeleteCheckInterval` with `hardDeleteTimeout/20` value